hazelcast-sql/pom.xml

```diff
@@ -121,13 +126,16 @@
                                     <excludes>
                                         <!-- Avoid automatic discovery of Calcite JDBC. -->
                                         <exclude>META-INF/services/java.sql.Driver</exclude>
+                                        <exclude>META-INF/MANIFEST.MF</exclude>
```

Comment:
@devozerov can we apply this for all artifacts, when I apply it to all artifacts I cannot see `MANIFEST.MF` though.

Reply:
Currently, there are still warnings about it MANIFEST.MF overlap when building the `sql` module. Another thing is that packages from the `sql` module are not included in the resulting MANIFEST.MF. I am not sure how it should work in the end, but I would try to do the following for both `fat-shaded-jar` and `fat-jar` goals:
1. Exclude `MANIFEST.MF` from all packages (`<artifact>*:*</artifact>`)
1. Include `MANIFEST.MF` for the hazelcast-sql artifact
1. Make sure that `sql` manifest is merged into the final manifest using `ManifestResourceTransformer` [1]. I do not know where exactly this transformer should be defined, unfortunately

This way we will not have warnings and at the same time the final MANIFEST.ML will include packages from the `sql` module.

But I would also ask someone with more experience with Maven on how it should work in the end.

[1] https://maven.apache.org/plugins/maven-shade-plugin/examples/resource-transformers.html#ManifestResourceTransformer

Reply:
@devozerov updated with targeted filters, I could not make it possible with `ManifestResourceTransformer` FYI. PTAL.